### ISSUE TITLE
Wrong "name" parameter default for Nadam optimizer

### ISSUE THREAD

**Issue by jpatts** [tensorflow/tensorflow#37179]
## URL(s) with the issue:
https://www.tensorflow.org/api_docs/python/tf/keras/optimizers/Nadam

## Description of issue (what needs changing):
Optional name attribute is said to default as "Adamax" rather than "Nadam"

### Clear description

name: Optional name for the operations created when applying gradients. Defaults to "Adamax".

### Correct links

https://github.com/tensorflow/tensorflow/blob/v2.1.0/tensorflow/python/keras/optimizer_v2/nadam.py#L33-L238

Source code correctly shows default as name="Nadam". Therefore documentation issue.

### Submit a pull request?

Not at the moment

**Comment by ashutosh1919** [comment]
Please close this issue. The error was resolved in #37111
